app/assets/javascripts/discourse/app/templates/mobile/components/categories-only.hbs

```diff
@@ -1,9 +1,17 @@
 {{#if this.categories}}
   {{#if this.filteredCategories}}
     <div class="category-list {{if this.showTopics 'with-topics'}}">
+      <PluginOutlet
+        @name="before-mobile-categories"
+        @outletArgs={{hash categories=this.filteredCategories}}
+      />
       {{#each this.filteredCategories as |c|}}
         <ParentCategoryRow @category={{c}} @showTopics={{this.showTopics}} />
       {{/each}}
+      <PluginOutlet
+        @name="after-mobile-categories"
+        @outletArgs={{hash categories=this.filteredCategories}}
+      />
```

Comment:
I think we can use a  a single outlet here? (that should give `__before` and `__after` connectors)

```suggestion
      <PluginOutlet
        @name="mobile-categories"
        @outletArgs={{hash categories=this.filteredCategories}}
      >
        {{#each this.filteredCategories as |c|}}
          <ParentCategoryRow @category={{c}} @showTopics={{this.showTopics}} />
        {{/each}}
      </PluginOutlet>
```